core/lib/generators/spree/install/templates/config/initializers/spree.rb

```diff
@@ -36,6 +36,18 @@
 
   # Custom logo for the admin
   # config.admin_interface_logo = "logo/solidus_logo.png"
+
+  # Gateway credentials can be configured statically here and referenced from
+  # the admin. They can also be fully configured from the admin.
+  #
+  # config.static_model_preferences.add(
+  #   Spree::Gateway::StripeGateway,
+  #   'example_credentials',
```

Comment:
This is a little ambiguous. When I first read it, I thought it meant to put my credentials here. Not that this is a nice name for referencing these credentials later. Maybe something like `stripe_env_credentials` would be clearer.